src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -319,7 +338,11 @@ class Task extends React.Component<Props, State> {
                         </div>
                         <div className="bcs-Task-status">
                             <TaskStatus status={status} />
-
+                            {isMultiFile && (
+                                <LabelPill.Pill data-testid="multifile-badge">
```

Comment:
In the same vein as https://github.com/box/box-ui-elements/pull/2049/files#r398273375, what about either pulling the logic out of the TaskCompletionRuleIcon & and TaskStatus helpers, or making a helper component that behaves similar to those?

This does its own visibility check: https://github.com/box/box-ui-elements/blob/0a6eeec68da9869e0a8238c90d52108fdd3a3747/src/elements/content-sidebar/activity-feed/task-new/TaskCompletionRuleIcon.js#L19

Reply:
Ah, I'll put it in its own component since the inbox uses the same icon as well